### PR TITLE
Add install instructions

### PR DESCRIPTION
**Pull request by 0323pin**
0.4.1 merged a few hours ago, https://mail-index.netbsd.org/pkgsrc-changes/2022/10/28/msg262617.html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,6 +126,12 @@ Wthrr is published as a crate on [crates.io](https://crates.io/crates/wthrr) and
 cargo install wthrr
 ```
 
+For NetBSD users there's a package available from the [official repositories](https://pkgsrc.se/misc/wthrr).
+
+```
+pkgin install wthrr
+```
+
 Requirements and other installations methods can be found in [`INSTALL.md`](https://github.com/tobealive/wthrr-the-weathercrab/blob/main/INSTALL.md).
```
Comment (tobealive) [inline README.md:135]: As the readme got a bit bloated, e5f85a1cccea93f3756af0ae561f62ad4bfcf917 moved more installation instructions - also for NetBSD - to [INSTALL.md](https://github.com/tobealive/wthrr-the-weathercrab/blob/main/INSTALL.md#installation) 

Kind of like the alacritty repo is doing it.

Do they have for some reason be in the README.md for NetBSD?

Comment (0323pin) [reply README.md:135]: Nope, there's no reason.
Please feel free to have it in INSTALL.md :)

Comment (0323pin) [reply README.md:135]: Ops ... 😮 
Sorry, it seems like this is a repated PR 🤦‍♂️

Comment (tobealive) [reply README.md:135]: Yes, no worries 🙋🏽‍♂

Ok, let's stay in the new structure then.

**Pull request closed without merging**